src/components/auth/RegisterChildren.vue

```diff
@@ -545,6 +549,7 @@ function codeNotRight(index) {
 function deleteStudentForm(student) {
   if (state.students.length > 1) {
     state.students.splice(student, 1); // Remove the student at the specified index
+    showConsent.value.pop();
   } else {
     alert('At least one student is required.'); // Prevent deleting the last student form
     submitted.value = false;
```

Comment:
```suggestion
function deleteStudentForm(index) {
  if (state.students.length > 1) {
    state.students.splice(index, 1); // Remove the student at the specified index
    showConsent.value.splice(index, 1)
  } else {
    alert('At least one student is required.'); // Prevent deleting the last student form
    submitted.value = false;
```
A bit more clarity here, since `deleteStudentForm()` takes an integer as an argument rather than a student object.